### PR TITLE
Update visibility icons on keep cards and library headers

### PR DESCRIPTION
**Pull request by DerekBurch**
This removes the background colors on library visibility in the library header, renames 'organization private' to 'team' and removes the red circle behind the private lock on library cards.

https://app.asana.com/0/20751731968782/51714360963530

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.styl [hunk 1]
```diff
@@ -404,14 +403,12 @@ followerPicDiam = 2.2em
     margin-right: 5px
 
 .kf-lh-org-private
-  background: readableYellowColor
 
   &::after
-    content: 'Organization Private'
+    content: 'Team'
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.styl:408]: If your reaction to this was, "Oh, it's weird to use CSS to choose text". Yes, yes it is.

**Comment by andrewconner** [comment]
-7 lines, a very good commit

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.styl (hunk 1 above)
Comment (DerekBurch) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.styl:408]: Should most strings just be in the template files then? (For when changing things in the future)

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryHeader.styl:408]: I'd err towards yes... @cvializ probably has a better informed opinion. CSS is just usually the last place I look to change text. It's more readable (for me, at least) just to have it where it happens in the template. But I may be behind the times

**Pull request merged**